### ISSUE TITLE
Energy cubes and some generators bad influence at TPS

### ISSUE THREAD

**Issue by Radviger** [aidancbrady/Mekanism#2449]
Hi, @aidancbrady!
On my server where is above 90 mods, the most laggy mod is Mekanism. I'm not trying to insult you, but your mod haven't optimization and this problem corrupts such an interesting and progressive mod. I think that without optimizing your mod just die out. Most of server administrators that I know says that they don't like mekanism because of lags.
When my server more than 30 players its start lagging and opis says me that the most luggy tiles is from mekanism, such as energy cubes, windmill, solar generators, etc. And because most of players cannot optimize their bases, server just slowly die. It's really terrible.
I really hope that you well think about this and make the right decision. Thanks!

**Comment by Greylocke** [comment]
Which version of Mekanism are you using? What are the mean and total times for the top Mekanism TEs?

**Comment by aidancbrady** [comment]
Yeah - I can't do much about this without any data.  Most server owners I've spoken with have seen a great increase in performance with v8.

**Comment by unpairedbracket** [comment]
Exactly - we do actually work hard to try and optimise things. I spent many hours rewriting the recipe system, to hopefully improve performance, in the lead-up to v8, and in fact I am currently (and have been for the past week) rewriting the whole transmitter network system in the hope that it'll make us a lot less hard-going on chunk loads (a merge I'm not looking forward to). It's just that we've got such a large mod on our hands at this point that it makes it difficult to keep on top of every possible optimisation issue.
